style.css

```diff
@@ -20,7 +20,7 @@ input{
     border-radius: 4px;
     vertical-align: middle;
     box-sizing: border-box;
-    color: #443729;
+    color: #ccbfb1;
```

Comment:
i dont think so , dont teach me what a good color is!! did u understand???!!#$@%36